src/sssom/cli.py

```diff
@@ -253,14 +254,25 @@ def split(input: str, output_directory: str):
 @input_argument
 @output_option
 @click.option("-W", "--inverse-factor", help="Inverse factor.")
-def ptable(input, output: TextIO, inverse_factor):
+@click.option(
+    "--default-confidence / --no-default-confidence",
+    default=False,
+    is_flag=True,
```

Comment:
1. Dont make it a flag, just make it 

```
--confidence X
```

where X is a number, e.g.

```
--confidence 0.9
```

Much more flexible, especially if you deal with mapping sets you trust differently!

Also, I thought about it and we can catch two flies with one here: Lets say this is the confidence of the person processing the mapping. We can use this not only to fill in missing confidence, but perhaps (in a second step, not this PR) to re-calculate existing confidence values.

So lets say we have a mapping that already has 0.8 confidence. If the mapping user provides `--confidence 0.9` then we could recalcularte the confidence like this `0.8*0.9=0.71` I think that is useful for many use cases..

Reply:
So in other words if an external confidence is provided (say 0.9 in your example) and existing confidence reduces (0.72)?

Reply:
I can do recalculation in this PR itself, it's very simple!